_test_/likes.test.js

```diff
@@ -0,0 +1,26 @@
+/*eslint-disable */
+
+
+import { getCommentCount } from "../modules/likesCounter";
```

Comment:
You have done great by including tests for the like counter. However, the tests seem not to be passing. Kindly fix it.

![image](https://github.com/wolterbwambale/javascriptCapstone/assets/20554583/dc1098b9-0311-43d6-88cc-82376bca0bf4)